ast/ast.go

```diff
@@ -53,28 +53,28 @@ type Statement interface {
 func (QueryStatement) isStatement()     {}
 func (CreateDatabase) isStatement()     {}
 func (CreateTable) isStatement()        {}
-func (CreateSequence) isStatement()     {}
-func (CreateView) isStatement()         {}
-func (DropView) isStatement()           {}
 func (CreateIndex) isStatement()        {}
-func (CreateVectorIndex) isStatement()  {}
+func (CreateView) isStatement()         {}
+func (CreateChangeStream) isStatement() {}
 func (CreateRole) isStatement()         {}
+func (CreateSequence) isStatement()     {}
+func (CreateVectorIndex) isStatement()  {}
 func (AlterTable) isStatement()         {}
 func (AlterIndex) isStatement()         {}
+func (AlterChangeStream) isStatement()  {}
 func (AlterSequence) isStatement()      {}
 func (DropTable) isStatement()          {}
```

Comment:
Comment in PR says

> This order follows the official documentation:

but this ordering sorts `Create`, `Alter`, `Drop` first, and object type second.

Official document ordering is object type first, `Create`, `Alter`, `Drop` second.

Reply:
After one night, I think it is better to use the official doc order directly because we can quickly check which statements are missing in memefish by comparing the official doc list.

@apstndb What do you think?

Reply:
I agree that the official doc order itself is better ordering because:

* I feel official documented order is natural for contributors.
* There are many DDLs without `CREATE`, `ALTER`, `DROP` prefix so I feel manipulation-based ordering is hard to define consistently.
* Usually, DDLs for the same object type are implemented in the same PR(or consective PRs), so diff will be more cunsecutive in object-based ordering.